### ISSUE TITLE
color tweet dots based on language

### ISSUE THREAD

**Issue by mef** [Mango-information-systems/twitto_be#107]
In the tweets map, dots could be colored based on the tweets language.

Advantages:

* more space-efficient in the dashboard than a chart of number of tweets per language
* gives an idea of the languages used in the tweets in each region of Belgium (multidimensional :))

Languages to show: NL, FR, EN, Other (DE is probably too rarely used, to be checked)

**Comment by mef** [comment]
linking related commits: 

* f20d795
* 4594541

**Comment by mef** [comment]
Storage of tweets language has been pulled inside branch `3.0.0-wip`. This means that once finalized, this feature can be deployed in production not earlier than 24 hours after the deployment of `v3.0.0`.

**Comment by mef** [comment]
Branch 3.0.0-pre, which includes the storage of tweets languages, has just  been deployed in production. This means that in 24 hours, this feature can be deployed.

**Comment by mef** [comment]
Heads-up @11digitlabs params.json now needs to have a `"colors"` property inside nested under `maps`.

````json
		, "colors": {
			"en": "rgba(0, 172, 237, .3)"
			, "fr": "rgba(0, 128, 0, .3)"
			, "nl": "rgba(255, 115, 0, .3)"
			, "other": "rgba(0, 0, 0, .3)"
		}
````

This determines the dot colors based on tweet's language.

Currently in branch issue-107-wip, may be merged inside 3.0.0-wip depending on deployment schedule / progress on the other issues.
